spec/helper/test_models.rb

```diff
@@ -12,18 +12,6 @@ class CopyableCoin < ActiveRecord::Base
 
 
 
-#*******************************************************************************
-#  For testing observers.
-#
-
-class CopyableCar < ActiveRecord::Base
```

Comment:
I looked at this code and I don't see this class being useful in testing anything.